code/modules/medical/autopsy.dm

```diff
@@ -156,18 +136,23 @@
 		scan_data += "Possible weapons:<br>"
 		for(var/weapon_name in weapon_chances)
 			scan_data += "\t[100*weapon_chances[weapon_name]/total_score]% [weapon_name]<br>"
-
 		scan_data += "<br>"
-
 		n++
-
 	if(chemtraces.len)
 		scan_data += "<b>Trace Chemicals: </b><br>"
 		for(var/chemID in chemtraces)
 			scan_data += chemID
 			scan_data += "<br>"
-
 	scan_data += "<br>[advanced_butchery]<br>"
+	return scan_data
+
+/obj/item/weapon/autopsy_scanner/verb/print_data()
+	set category = "Object"
+	set src in view(usr, 1)
+	set name = "Print Data"
+	if(usr.isUnconscious() || !(istype(usr,/mob/living/carbon/human)))
```

Comment:
instead, `if(usr.stat||!ishuman(usr))`

Reply:
part of old code preserved for generations, I'll update it a bit

Reply:
It's not bad to look at existing code as a starting point, but remember that most oldcode beyond say six years ago wouldn't pass modern review standards. That means if you paste, you're gonna get stuck with some modernizing in review.

That's generally a good thing though, since it means you are slowly erasing bad code